server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -269,17 +270,24 @@ private Modal renderUpdateNoteConfirmationModal(
       long programId, Application application, Http.Request request) {
     ButtonTag triggerButton =
         makeSvgTextButton("Edit note", Icons.EDIT).withClasses(AdminStyles.TERTIARY_BUTTON_STYLES);
+    // Give each form a unique id based on the application.
+    String formId = String.format("note-form-%d", application.id);
```

Comment:
If we're not selecting based on this ID anywhere, trying to make it human readable doesn't provide a lot of benefit in my opinion. It also makes it potentially more likely to run into conflicts if we were to either 1) copy/paste or 2) Have a similar element with the same name elsewhere in the DOM. Both of these are unlikely, obviously.

That said, if it doesn't need to have human-readable meaning, I'd prefer making that explicit through initializing the form with a random ID.

We could use a random UUID. It'd make sense in my opinion to pull Modal.randomModalId() into a helper function in something in app/views for this purpose.

Reply:
Done. Modal is already here so used it directly.